src/__tests__/media-query-test.js

```diff
@@ -210,15 +211,27 @@ describe('Media query tests', () => {
       }
     }
 
+    @Radium({matchMedia})
+    class TestComponent extends Component {
```

Comment:
I think this is fine.

Couldn't this be abstracted into a function so that all future tests requiring a StyleRoot would get wrapped automatically? Or is that not possible with TestUtils?